README.md

```diff
@@ -1,2 +1,28 @@
+![Build Status](https://img.shields.io/github/actions/workflow/status/omnivector-solutions/vantage-agent-charms/test.yaml?branch=main&label=build&logo=github&style=plastic)
+![GitHub Issues](https://img.shields.io/github/issues/omnivector-solutions/vantage-agent-charms?label=issues&logo=github&style=plastic)
+![Pull Requests](https://img.shields.io/github/issues-pr/omnivector-solutions/vantage-agent-charms?label=pull-requests&logo=github&style=plastic)
+![GitHub Contributors](https://img.shields.io/github/contributors/omnivector-solutions/vantage-agent-charms?logo=github&style=plastic)
+
+> An [Omnivector](https://www.omnivector.io/) initiative
+>
+> [![omnivector-logo](https://omnivector-public-assets.s3.us-west-2.amazonaws.com/branding/omnivector-logo-text-black-horz.png)](https://www.omnivector.io/)
+
 # vantage-agent-charms
-The collection of charms used to integrate any Slurm cluster with Vantage.
+
+This repo provides the charms used to integrate any Slurm cluster with Vantage. Each
+charm deploys the corresponding agent's snap package and manages it's life-cycle in the
```

Comment:
```suggestion
charm deploys the corresponding agent's snap package and manages its lifecycle in the
```

Reply:
Good catch on "it's".

I looked it up: "life cycle" is preferred over "life-cycle" or "lifecycle".

Reply:
Resolved in commit e15fece.